app/src/main/java/com/napp/napp/ui/login/bookingpage.java

```diff
@@ -0,0 +1,6 @@
+package com.napp.napp.ui.login;
+
+import android.app.Activity;
+
+public class bookingpage extends Activity {
```

Comment:
Is there a reason there are two booking and welcome pages? Also since they're not related to the login functionality, it might be better to split them out into their own folders

Reply:
Also, there seem to be no changes made to `AndroidManifest.xml`. You'll need to add entries for these activities in there, similar to what you did with the faq page [here](https://github.com/00Chaotic/nApp/pull/9/files#diff-7fa6aef292187a049f7a4d6060d8df3ba212d838789c78940bd363344b1c38cd). This is needed if we want to actually be able to visit those pages.